docs/components/motor/gpio/encoded-motor.md

```diff
@@ -113,7 +118,12 @@ Here’s an example configuration:
           "dir": "18"
         },
         "encoder": "myEncoder",
-        "ticks_per_rotation": 9600
+        "ticks_per_rotation": 9600,
+        "control_parameters": {
+          "p": 1,
+          "i": 2,
+          "d": 3
```

Comment:
@martha-johnston are there any more realistic-looking values we might want to put here?

Reply:
maybe p: 0.2, i: 0.5, d: 0.0

Reply:
but it can vary a lot depending on the system

Reply:
that's okay, i think having values that make sense for some system is better than totally dummy values